EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: <to be assigned>
+title: JSON-RPC method to sign Ethereum messages (eth_signMessage)
+author: Pedro Gomes <@pedrouid>
+discussions-to: <to be assigned>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-03-10
+requires: 191, 1474
+---
+
+## Simple Summary
+Introduces a new JSON-RPC method name to request Ethereum message signatures in light of eth_sign and personal_sign history
```

Comment:
I'm not sure what "in light" means here?

Reply:
Basically "considering" or "informed by".

Reply:
Oh okay I see. I recommend using rewording to more clearly explain the relationship between this RPC method and the two referred to here.